worlds/ff1/Items.py

```diff
@@ -27,6 +26,13 @@ class ItemData(NamedTuple):
     "EarthOrb", "FireOrb", "WaterOrb", "AirOrb"
 ]
 
+FF1_USEFUL_LIST = [
+    "Tail", "Masamune", "Xcalber", "Katana", "Vorpal",
+    "OpalArmor", "DragonArmor", "Opal",
+    "OpalShield", "AegisShield", "OpalHelm", "Ribbon",
+    "OpalGauntlets",
```

Comment:
OpalArmor, OpalShiled, OpalGauntlets and OpalHelm are all historically considered "whammy" legendary gear. Including these really depends on what this will be used for. If you want a high amount of density of really good gear I would exclude it. If it is more like a "rare"  card where some are great and some are only okay then it might be better to remove

Reply:
E.g. I hate doing 1500 Archery in OOT, so I exclude it. If these items are "Useful" then they will never appear on my 1500 archery, because i aint doin it. lol

Reply:
I assume in that case having this list be relatively small would be better. In that case I would recommend removing the opal gear (except for Opal which is the Opal bracelet which is a really good piece of gear). But ultimately up to Alchav

Reply:
Definitely shouldn't be ultimately up to me. I'm a noob at this game and just knew that the Tail should definitely not be excluded and grabbed some other item names from lists of "legendary" items from the FF1R code. I'll take out the other Opal items.